DXMainClient/DXGUI/Multiplayer/GameLobby/CnCNetGameLobby.cs

```diff
@@ -151,9 +152,25 @@ public override void Initialize()
             tunnelSelectionWindow.CenterOnParent();
             tunnelSelectionWindow.Disable();
 
+            btnChangeTunnel = new XNAClientButton(WindowManager);
+            btnChangeTunnel.Name = "btnChangeTunnel";
+            btnChangeTunnel.ClientRectangle = new Rectangle(btnLeaveGame.Right - btnLeaveGame.Width - 145,
+                btnLeaveGame.Y, 133, 23);
+            btnChangeTunnel.Text = "Change Tunnel";
+            btnChangeTunnel.LeftClick += BtnChangeTunnel_LeftClick;
+
+            btnChangeTunnel.Enabled = false;
+            btnChangeTunnel.Visible = false;
+
+            AddChild(btnChangeTunnel);
             WindowManager.AddAndInitializeControl(gameBroadcastTimer);
         }
 
+        private void BtnChangeTunnel_LeftClick(object sender, EventArgs e)
+        {
+            ShowTunnelSelectionWindow("Select tunnel server:");
+        }
```

Comment:
Since this function is so simple, it could be better to embed its functionality in a lambda expression on line 160. Typing it with an expression body is another option.

Reply:
Literally the function next to the one I added (`GameBroadcastTimer_TimeElapsed`) is also very simple and not declared as lambda or via expression body, so I think what you said should not be done in this PR as it would make code not match the style of the rest.